plugins/web/opentelemetry-instrumentation-long-task/src/types.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import type { Span } from '@opentelemetry/api';
+import type { InstrumentationConfig } from '@opentelemetry/instrumentation';
+
+// Currently missing in typescript DOM definitions
+export interface PerformanceLongTaskTiming extends PerformanceEntry {
+  attribution: TaskAttributionTiming[];
+}
+
+export interface TaskAttributionTiming extends PerformanceEntry {
+  containerType: string;
+  containerSrc: string;
+  containerId: string;
+  containerName: string;
+}
+
+export type ObserverCallback = (
+  span: Span,
+  entry: PerformanceLongTaskTiming
```

Comment:
Thanks for addressing all the comments :)
One last request: some hook functions started small and got bigger over time as more data was added to the function signature as arguments.
We usually try to avoid that by sending all the hook data as `info` object which can be extended in the future in a non-breaking fashion and keeps the function signature and implementation short and lean.
You can see example of that in `aws-sdk` [here](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/6ba387ab45d0f67fdfac85c43bb0f0f67f2a119b/plugins/node/opentelemetry-instrumentation-aws-sdk/src/types.ts#L44).

Reply:
That's a good point. Thanks for all the feedback on this! We're excited to see it in prod!